src/schema/responses.ts

```diff
@@ -106,6 +106,10 @@ export const GetChannelPermissionsResponse = zod.object({
   permissions: zod.bigint().or(zod.string()),
 });
 
+export const OpenExternalLinkResponse = zod.object({
+  opened: zod.boolean(),
+});
+
```

Comment:
general question: how does this behave against a version of the client that doesn't send the `opened` flag across RPC? my gut is that since this isn't marked as optional and we're using `.parse()` it would error out, but I also don't know if we have error handling for that buried elsewhere (I'm assuming we do since otherwise I imagine this would have been a problem already).

Reply:
would defaulting this to `false` be appropriate?

Reply:
Good call. I've updated this to use `fallbackToDefault` and return `{ opened: null }` in the case where parsing fails (ie: discord clients that don't support this response). I've also [updated the documentation](https://github.com/discord/discord-api-docs/pull/7273/commits/7251a1c8de84457617e40f9499cff7db6bdbaf78) with the null case and a note on when it occurs.